packages/js-sdk/src/datastore/networkstore.ts

```diff
@@ -270,16 +252,20 @@ export class NetworkStore {
     return response.data;
   }
 
-  save(docs: any, options?: any) {
-    if (!isArray(docs)) {
-      const kmd = new Kmd(cloneDeep(docs));
+  save(doc: any, options?: any) {
+    const apiVersion = getApiVersion();
+    if (apiVersion >= 5 && isArray(doc)) {
+      throw new KinveyError('Unable to save an array of entities. To insert multiple entities, please use "create" function.');
```

Comment:
Nit:
Think calling `create` a method is better, since it's a method of a class, not a standalone function.